doc/cloud_init.rst

```diff
@@ -3,106 +3,39 @@
-            --attach commissaire.txt:x-commissaire-host \
-            > user-data
-
-.. note::
-
-   The content type of the ``commissaire.txt`` config file must be
-   ``text/x-commissaire-host`` (the attach option assumes the "text/" part).
+Let's say you have the following properties:
 
-Alternatively, you can instruct cloud-init to download the ``part-handler.py``
-script from a URL at boot time by attaching a ``text/x-include-url`` file:
-
-.. code-block:: shell
-
-   $ python make-mime.py \
-            --attach cloud-config.txt:cloud-config \
-            --attach include.txt:x-include-url \
-            --attach commissaire.txt:x-commissaire-host \
-            > user-data
-
-.. note::
-
-   Whether the part handler is included by way of direct embedding or a URL,
-   it must appear in the multipart file `before` the ``text/x-commissaire-host``
-   part so cloud-init knows how to handle the ``text/x-commissaire-host`` part.
+* A Commissaire username of USER
+* A Commissaire password of PASS
+* A Commisaire cluster you want new hosts to join called CLUSTER
+* A Commissaire REST Server listening at https://example.com/
+* The expectation of having the ``user-data`` file at ``./CLUSTER.userdata``
 
-
-Example: User-Data Creation
----------------------------
-This example shows how to create the ``user-data`` file which can be passed
-on boot via your OS/cloud provider. See your OS/cloud provider for specific
-details on how to pass the user-data file properly.
+You would create the ``user-data`` file like so:
 
 .. code-block:: shell
 
-  $ wget "http://bazaar.launchpad.net/~cloud-init-dev/cloud-init/trunk/download/head:/makemime.py-20130122042101-yeyh09eokb87l6oa-1/make-mime.py"
-  # <snip>
-  2016-03-24 10:54:33 (167 MB/s) - ‘make-mime.py’ saved [1946/1946]
-
-  $ wget "https://raw.githubusercontent.com/projectatomic/commissaire/master/tools/cloud-init/part-handler.py"
-  # <snip>
-  2016-03-24 10:56:33 (772 MB/s) - ‘part-handler.py’ saved [4848/4848]
-
-  $ wget "https://raw.githubusercontent.com/projectatomic/commissaire/master/tools/cloud-init/commissaire.txt"
-  # <snip>
-  2016-03-24 10:57:07 (752 MB/s) - ‘commissaire.txt’ saved [866/866]
-
-  $ wget "https://raw.githubusercontent.com/projectatomic/commissaire/master/tools/cloud-init/cloud-config.txt"
-  # <snip>
-  2016-03-24 10:59:01 (677 MB/s) - ‘cloud-config.txt’ saved [48/48]
-
-  $ $EDITOR cloud-config.txt     # Edit the cloud config to your liking
-  $ $EDITOR commissaire.txt      # Edit the commissaire config to your liking
-  $ python make-mime.py \
-         --attach cloud-config.txt:cloud-config \
-         --attach part-handler.py:part-handler \
-         --attach commissaire.txt:x-commissaire-host \
-         > user-data
-  WARNING: content type 'text/x-commissaire-host' for attachment 3 may be incorrect!
-  $
+   $ commctl user-data \
+     --password \
+     --username USER \
+     --cluster CLUSTER \
+     --endpoint https://example.com/ \
+     CLUSTER.userdata
```

Comment:
The invocation will be slightly different if you take my suggestion in https://github.com/projectatomic/commctl/pull/33.
```
--output CLUSTER.userdata  or  > CLUSTER.userdata
```